openedx/features/content_tagging/tests/test_rules.py

```diff
@@ -60,23 +64,45 @@ def setUp(self):
             username="user_org2",
             email="staff+org2@example.com",
         )
-        update_org_role(
-            self.staff, OrgContentCreatorRole, self.user_org2, [self.org2.short_name]
-        )
+        update_org_role(self.staff, OrgContentCreatorRole, self.user_org2, [self.org2.short_name])
 
         # Normal user: no course creator access
         self.learner = User.objects.create(
             username="learner",
             email="learner@example.com",
         )
 
+    # ToDo: remove old permission code
+    # def _expected_users_have_perm(self, perm, obj, learner_perm=False, learner_obj=False, user_org2=True):
+    #     """
+    #     Checks that all users have the given permission on the given object.
+
+    #     If learners_too, then the learner user should have it too.
+    #     """
+    #     # Global Taxonomy Admins can do pretty much anything
+    #     assert self.superuser.has_perm(perm)
+    #     assert self.superuser.has_perm(perm, obj)
+    #     assert self.staff.has_perm(perm)
+    #     assert self.staff.has_perm(perm, obj)
+    #     assert self.user_all_orgs.has_perm(perm)
+    #     assert self.user_all_orgs.has_perm(perm, obj)
+
+    #     # Org content creators are bound by a taxonomy's org restrictions
+    #     assert self.user_both_orgs.has_perm(perm) == learner_perm
+    #     assert self.user_both_orgs.has_perm(perm, obj)
+    #     assert self.user_org2.has_perm(perm) == learner_perm
+    #     # user_org2 does not have course creator access for org 1
+    #     assert self.user_org2.has_perm(perm, obj) == user_org2
+
+    #     # Learners can't do much but view
+    #     assert self.learner.has_perm(perm) == learner_perm
+    #     assert self.learner.has_perm(perm, obj) == learner_obj
+
     def _expected_users_have_perm(
-        self, perm, obj, learner_perm=False, learner_obj=False, user_org2=True
+        self, perm, obj, learner_perm=True, learner_obj=False, user_org2=True
     ):
         """
-        Checks that all users have the given permission on the given object.
-
-        If learners_too, then the learner user should have it too.
+        Checks that all users have the given permission on the given taxonomy.
```

Comment:
nit: this isn't just about taxonomy perms:
```suggestion
        Checks that all users have the given permission on the given object.
```

Reply:
Reverted this change